maven-extension/src/main/java/io/opentelemetry/maven/OtelLifecycleParticipant.java

```diff
@@ -43,6 +58,10 @@ public final class OtelLifecycleParticipant extends AbstractMavenLifecyclePartic
    */
   @Override
   public void afterProjectsRead(MavenSession session) {
+    registerExecutionListener(session);
```

Comment:
Why did we change something here? I didn't expect this PR to touch the execution listener. Did I miss something? I'll look at the code.